File: API/Models/Product.cs
using System.Collections.Generic;

namespace API.Models
{
    public class Product
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Category { get; set; }
        public string Brand { get; set; } 
        public double Price { get; set; }
        public string Details { get; set; }
        public Photo Photo { get; set; }
        public List<Size> Sizes { get; set; }
    }
}

File: client/Models/product.ts
import {Photo} from "../Models/photo"
import {Size} from "../Models/size"
export interface Product
{
    id:       number;
    name:     string;
    category: string;
    brand:    string;
    price:    number;
    details:  string;
    photo:    Photo;
    sizes:Array<Size>;
    pupulerItems:boolean;
}


